tests/basic/storage_pool_test.go

```diff
@@ -9792,3 +9792,94 @@ var _ = Describe("{PoolExpandRebalanceShutdownNode}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{AddDriveWithKernelPanic}", func() {
+	//1) Deploy px with cloud drive.
+	//2) Create a volume on that pool and write some data on the volume.
+	//3) Expand pool by adding cloud drives.
+	//4) Inject kernel panic where the pool expansion is in-progress
+	//5) Verify total pool count after addition of cloud drive with kernel panic
+
+	var testrailID = 0
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("AddDriveWithKernelPanic", "Initiate pool expansion using add-drive and do kernel panic while it is in progress", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+
+	stepLog := "should get the existing storage node and expand the pool by adding a drive"
+
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("pladddrvwrst-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get Pool with running IO on the cluster
+		poolUUID, err := GetPoolIDWithIOs(contexts)
+		log.FailOnError(err, "Failed to get pool running with IO")
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+
+		poolToBeResized, err := GetStoragePoolByUUID(poolUUID)
+		log.FailOnError(err, "failed to get pool from pool uuid [%v]", poolUUID)
+		
+		stNode, stNodeerr := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(stNodeerr, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+
+		expectedSize := (poolToBeResized.TotalSize / units.GiB) + 200
+		log.InfoD("Current Size of the pool %s is %d", poolUUID, poolToBeResized.TotalSize/units.GiB)
+		
+		expanderr := Inst().V.ExpandPool(poolUUID, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK, expectedSize, true)
+		log.FailOnError(expanderr, "Failed to initiate expand on pool [%v]", poolUUID)
+
+		isjournal, journalerr := IsJournalEnabled()
+		log.FailOnError(journalerr, "failed to get journal disk details")
+		
+		time.Sleep(5 * time.Second)
+		cmd := "echo c > /proc/sysrq-trigger"
+
+		// Execute the command to generate kernel panic
+		log.Infof("Executing command on node, [%v]", stNode.Name)
+		_, err = Inst().N.RunCommandWithNoRetry(*stNode, cmd, node.ConnectionOpts{
+			Timeout:         2 * time.Minute,
+			TimeBeforeRetry: 10 * time.Second,
+		})
+		
+		re, _ := regexp.Compile(".*remote command exited without exit status or exit signal")
+		regMatch := re.MatchString(fmt.Sprintf("%v", err))
+		dash.VerifyFatal(regMatch, true, " force panic the node successful?")
+		
+		err = Inst().N.TestConnection(*stNode, node.ConnectionOpts{
+			Timeout:         15 * time.Minute,
```

Comment:
How are these heuristics determined? 
In general we don't love hardcoded values. Would it make sense to define some constants?

Reply:
@hramadoss-px I think already some constants are defined. you can use them.

Reply:
Modified,

err = Inst().N.TestConnection(*stNode, node.ConnectionOpts{
			Timeout:         addDriveUpTimeOut,
			TimeBeforeRetry: 10 * time.Second,
		})